packages/templating-tools/package.js

```diff
@@ -16,7 +16,7 @@ Package.onUse(function(api) {
     // boilerplate-generator uses spacebars-compiler.)
     // XXX maybe uglify should be applied by this plugin instead of via magic
     // weak dependency.
-    'minifier-js@1.2.14'
+    'minifier-js@1.2.14 || 2.0.0'
```

Comment:
Hmm, this does seem to be defeating the purpose of the original change.  Did I miss the discussion leading to this change somewhere?

@sethmurphy18 The original intention of #234 was to get [this change](https://github.com/meteor/blaze/pull/234/commits/d2ee1ac5ca2f9e7b943699e31924b316d1cbc954) included because the new minifier would actually minify in a different way.

Looks like that was reverted with https://github.com/meteor/blaze/pull/234/commits/5594d8095c81ea1bb09d66914814a5e725d012ff  and that possibly the only reason that Blaze is passing this is because we're explicitly pinning it to the current `1.2.14` version via this line (`2.0.0` is not published yet)?

I understand the changes to `spacebars-compiler` in order to get `_beautify` working and I think the direct dependency on the `uglify-js` NPM is fine for that, but what happened to the `templating-tools` change to compensate for the Babili minification?  Don't we still need that when the official `minifier-js` (the `2.0.0` in this version constraint) is published?